src/Components/__test__/App.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import App from '../../App';
+import store from '../../redux/configureStore';
+import '@testing-library/jest-dom';
+
+it('Check if Item component has changed', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>,
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it('Check if the App container is there', async () => {
+  render(
+    <Provider store={store}>
+      <BrowserRouter>
+        <App />
+      </BrowserRouter>
+    </Provider>,
+  );
+  const container = await screen.findByTestId('app-container');
+  expect(container).toBeInTheDocument();
+});
```

Comment:
- [ ] Some of your tests are failing. From the error message, for example, this particular file has a test that does not run at all. Kindly look into that and fix them.

  ![ryrut](https://user-images.githubusercontent.com/54848363/196728886-d7efe57d-aa1f-48e1-b57c-eae305ca7dfc.jpg)